CRM/Core/Block.php

```diff
@@ -440,6 +440,7 @@ private static function setShortcutValues($short) {
     }
     $value['title'] = $short['title'];
     $value['ref'] = $short['ref'] ?? '';
+    $value['shortCuts'] = [];
```

Comment:
In general I think this is the 'right' way- ie we should ensure keys are consistently assigned - but it's hard to do in bulk

Reply:
yeh I agree and also sometimes unit tests may short cut things that are added in the invoke routine as well